terraform-modules/cid-dashboards/README.md

```diff
@@ -0,0 +1,151 @@
+# CID Terraform Module: cid-dashboards
+
+Terraform module to deploy CID dashboards. This module is a wrapper around CloudFormation
+to allow you to deploy CID dashboards using your existing Terraform workflows. Under the
+hood, the module will deploy a CloudFormation stack which will provision the necessary
+resources and a custom Lambda function to create the dashboards using `cid-cmd`.
+
+## Before You Start
```

Comment:
Can we point to official location? 

https://aws-managed-cost-intelligence-dashboards.s3.amazonaws.com/cfn/cur-aggregation.yaml

https://aws-managed-cost-intelligence-dashboards.s3.amazonaws.com/cfn/cid-cfn.yaml